backend/api/__init__.py

```diff
@@ -0,0 +1,25 @@
+from flask import (jsonify, Blueprint)
+from flask_restful import Api, Resource
+
+from ..models import Shop
+
+
+api_bp = Blueprint('api_bp', __name__)
+api = Api(api_bp)
+
+def as_dict(instance):
+    ''' Transform a model instance into a dict '''
+    return {
+        c.name: getattr(instance, c.name)
+        for c in instance.__table__.columns
+    }
+
+class Shops(Resource):
+    def get(self):
+        all_saved_shops = [as_dict(shop) for shop in Shop.query.all()]
+        if all_saved_shops:
+            return jsonify(all_saved_shops)
```

Comment:
We should always return a dict with at least `status` inside. So this would become `jsonify(status='success', shops=all_saved_shops)`

And I would just return an empty list, not make a special case for no shops.